dynawo/sources/API/CRT/CRTCriteriaImpl.h

```diff
@@ -59,11 +65,25 @@ class Criteria::Impl : public Criteria {
    * @copydoc Criteria::end() const
    */
   component_id_const_iterator end() const;
+
+  /**
+   * @brief Test if this criteria is limited to a specific country
+   * @param country to test
+   * @return @b true if this criteria should be limited to this country
+   */
+  bool containsCountry(const std::string& country) const;
```

Comment:
use a copydoc to avoid duplicating the doxygen doc

Reply:
I have to copy it as the copydoc does not work with some versions of doxygen when there is a const

Reply:
Ok.